src/SAWScript/Crucible/LLVM/X86.hs

```diff
@@ -607,31 +633,37 @@ setupSimpleLoopFixpointFeature ::
   C.CFG ext blocks init ret ->
   C.GlobalVar C.LLVM.Mem ->
   TypedTerm ->
-  IO (C.ExecutionFeature p sym ext rtp)
+  IO (C.ExecutionFeature p sym ext rtp, IORef (Crucible.LLVM.Fixpoint.ExecutionFeatureContext sym 64 ext))
 
-setupSimpleLoopFixpointFeature sym sc sawst cfg mvar func =
-  Crucible.LLVM.Fixpoint.simpleLoopFixpoint sym cfg mvar fixpoint_func
+setupSimpleLoopFixpointFeature sym sc sawst cfg mvar func = do
```

Comment:
Crucible only understands what4, not saw-core, so this function: takes the what4-internals inputs, converts to saw-core, evaluates the results, then converts back to what4.  This is used because saw-core is NOT a dependency of crucible, and therefore crucible is not capable of performing the saw-core concepts here.